sunpy/net/jsoc/jsoc.py

```diff
@@ -323,7 +324,7 @@ def search(self, *query, **kwargs):
         """
 
         return_results = JSOCResponse()
-        query = and_(*query)
+        query = and_(*(*query, attrs.Quality('?QUALITY >= 0?')))
```

Comment:
I think adding this here will cause errors if the user tries to specify their own Quality attr. See my comment further down about where I think the default would be best put.